rfcs/README.md

```diff
@@ -79,7 +79,7 @@ The RFC numbering convention is: `<type>:<id>`. The list of types (`<type>`) is
 | 03-018 | Interoperation CorDapp API       | Format   | [Proposed](./formats/network/interop-cordapp.md)                |
 | 03-019 | Besu Interop Contract API        | Format   | [Proposed](./formats/network/besu-interop-contract.md)          |
 | 03-020 | Application SDK                  | Format   | [Proposed](./formats/network/application-sdk.md)                |
-
+| 03-021 | IIN DID Method                  | Format   | [Proposed](./formats/network/iin_method.md)                |
```

Comment:
How about we call this "DID Method for DLT Networks"? Because that's what we are proposing this new method for, right? Networks and their participants? Within the spec itself, you can talk about IINs, but the title I think should be more generic, to appeal to an audience that understands DIDs and DLTs but is not familiar with interoperability needs.